tf2/test/simple_tf2_core.cpp

```diff
@@ -34,31 +34,22 @@
 #include "tf2/exceptions.h"
 #include "tf2/time.h"
 
-typedef std::chrono::system_clock::time_point TimePoint;
-
-void seed_rand()
-{
-  //Seed random number generator with current microseond count
-  srand((unsigned int)std::chrono::system_clock::now().time_since_epoch().count());
-};
-
-void generate_rand_vectors(double scale, uint64_t runs, std::vector<double>& xvalues, std::vector<double>& yvalues, std::vector<double>&zvalues)
-{
-  seed_rand();
-  for ( uint64_t i = 0; i < runs ; i++ )
-  {
-    xvalues[i] = 1.0 * ((double) rand() - (double)RAND_MAX /2.0) /(double)RAND_MAX;
-    yvalues[i] = 1.0 * ((double) rand() - (double)RAND_MAX /2.0) /(double)RAND_MAX;
-    zvalues[i] = 1.0 * ((double) rand() - (double)RAND_MAX /2.0) /(double)RAND_MAX;
-  }
-}
-
-
-
```

Comment:
in that case can you make this in separate commits in the future. This way you can submit the same commit upstream.